src/Controller/GUIController.py

```diff
@@ -107,6 +136,17 @@ def open_new_patient(self):
         if confirmation_dialog == QMessageBox.Yes:
             self.open_patient_window.emit()
 
+    def open_image_fusion(self):
+        patient_dict_container = PatientDictContainer()
+        filepath = patient_dict_container.path
```

Comment:
One last bug (hopefully):
I think there's an inconsistency in the default directory of open image fusion window.
Steps to reproduce:
1. Open OnkoDICOM, choose any directory path (mine is /home/linh/Downloads/DICOM-RT-04). Open one study in that folder.
![Screenshot from 2021-09-08 20-38-18](https://user-images.githubusercontent.com/30374401/132495075-b689c359-1fc1-4cd5-b526-bce4e36bb825.png)
2. Click on Image Fusion icon, not choose anything and click Exit (I'm not sure if choose an image to fuse result in the same bug or not, but I didnt choose anything. You can see here the default path is .../DICOM-RT-04
![Screenshot from 2021-09-08 20-39-21](https://user-images.githubusercontent.com/30374401/132495377-ed804b9b-0b36-4805-a850-82ad4678eab0.png)

3. Click on open a new patient, choose a different patient (i choose RT-01
![Screenshot from 2021-09-08 20-42-15](https://user-images.githubusercontent.com/30374401/132495681-1157ecf4-0c30-41f7-ac3b-24a657267b30.png)

4. In the new window, choose Image Fusion icon again. Now you see the default directory is still .../DICOM-RT-04 while it's supposed to be the path to DICOM-RT-01
![Screenshot from 2021-09-08 20-43-53](https://user-images.githubusercontent.com/30374401/132495858-1599a6d9-1a9f-4439-a07c-0d0a38f023f6.png)

Please let me know if you can recreate the issue.
One small thing: after the Image Fusion Tab is created, i see that I'm still at DicomView tab an have to click Image Fusion Tab to go there. Can you make the Image Fusion tab automatically selected after image fusion is done?

Reply:
> I think there's an inconsistency in the default directory of open image fusion window.

You're right, I wrote the scan directory assuming the images would all be in the same directory above the images, however this is not the case. ImageFusionWindow now loads the default directory instead

> One small thing: after the Image Fusion Tab is created, i see that I'm still at DicomView tab an have to click Image Fusion Tab to go there. Can you make the Image Fusion tab automatically selected after image fusion is done?

Done, and good idea because that was annoying me as well haha